dgym/collection.py

```diff
@@ -524,7 +523,6 @@ def from_json(
         From SmilesClickChem: https://zenodo.org/record/4100676
         
         """
-        import pandas as pd
```

Comment:
(This was duplicated)

Reply:
There may be other imports than we can remove, but its outside of the scope of this change.
It'd be prioritary to set the test bed and reduce repetition first, and only then hunt these down.